DEV_GUIDE.md

```diff
@@ -26,6 +27,22 @@ This document gives a detailed breakdown of the various build processes and opti
   * [Rendering documentation](#rendering-documentation)
 <!-- TOC -->
 
+## Project structure
+
+The module structure is as follows:
+
+* `kroxylicious-bom` the Bill Of Materials used by developers to conveniently declare a consistent set of Kroxylicious dependencies.
+* `krpc-code-gen` a code generator used to build the API module.
+* `kroxylicious-api` the APIs needed to write a filter.
+* `kroxylicious-unit-test` test utilities for filter authors.
+* `kroxylicious` the proxy runtime platform for hosting filters, including the networking code etc, but lacking a `main()` method
+* `kroxylicious-test-tools` ???
+* `kroxylicious-app` provides a `main()` method for the proxy.
+* `kroxylicious-multitenant` a filter that implements multitenancy
+* `kroxylicious-schema-enforcement` a filter that implements schema enforcement
```

Comment:
I think it would be a good idea to adopt a convention for the names of modules containing filters. e.g. these could be `kroxylicious-filter-multitenant` and `kroxylicious-filter-schema-enforcement`.

Reply:
Actually I think we missed a trick when push them into the additional filters module. We should have updated the groupId to be `io.kroxylicious.filters` and then we could drop the `korxylicious-filter` prefix